Getting Started / Site: surface My Preferences as a way to find your site (#36626)
DataDog/documentation: content/en/getting_started/site/_index.md

## Access the Datadog site

> {{< img src="getting_started/site/site.png" alt="The site URL in your browser tab" style="width:40%" >}}

Reviewer: I know this screenshot isn't part of the changes you made, but I'm wondering if we can remove it now that the initial sentence has been deleted. Without the lead-in, it sort of looks out of place and without context.

Author: Removed the site.png screenshot in 11cac596f8 — agreed it was orphaned without the lead-in sentence.

| AP1     | `https://ap1.datadoghq.com` | `ap1.datadoghq.com` | Japan |
| AP2     | `https://ap2.datadoghq.com` | `ap2.datadoghq.com` | Australia |

> If you have a custom domain, e.g. `demo.datadoghq.com` you can also find your site listed at the top of the **My Preferences** page. This also works for custom domains.

Reviewer: ```suggestion
If you have a custom domain, such as `demo.datadoghq.com`, you can find your site listed at the top of the **My Preferences** page.
```

Author: Applied — wording suggestion committed in 11cac596f8.

{{< img src="getting_started/site/site-in-preferences.png" alt="The top of the My Preferences page in Datadog, showing the organization name and site URL" style="width:80%" >}}

> To navigate to **My Preferences**, click your profile avatar in the lower-left, then select **My Preferences** from the menu.

Reviewer: ```suggestion
To navigate to **My Preferences**, click your profile avatar in the lower-left corner, then select **My Preferences** from the menu.
```

Author: Applied — wording suggestion committed in 11cac596f8.
